standard/expressions.md

```diff
@@ -1217,7 +1217,7 @@ The run-time processing of a function member invocation consists of the followin
   - `M` is invoked.
 - Otherwise, if the type of `E` is a value-type `V`, and `M` is declared or overridden in `V`:
   - `E` is evaluated. If this evaluation causes an exception, then no further steps are executed. For an instance constructor, this evaluation consists of allocating storage (typically from an execution stack) for the new object. In this case `E` is classified as a variable.
-  - If `E` is not classified as a variable, or if `V` is not a readonly struct type ([§16.2.2](structs.md#1622-struct-modifiers)), and `E` is one of:
+  - If `E` is not classified as a variable, or if `V` is not a readonly struct type ([§16.2.2](structs.md#1622-struct-modifiers)) and the declaration of `M` does not include the `readonly` modifier (§16.4.12), and `E` is one of:
```

Comment:
It's a little odd to refer to "a readonly struct type" but then talk about the declaration of M "including the readonly modifier", both in the same sentence. Is this because we can't say "readonly method" as it could be a property or indexer? Would "readonly member" work?

In particular, I think that would be slightly helpful in terms of there being multiple places where the modifier can occur for properties - either at the property declaration itself, or the accessor declaration. If we haven't already defined "readonly member" or "readonly function member" perhaps we could do so, then use it here?

I'd be fine to merge as-is and maybe have a clarity follow-up for this though.